business_objects/upload_task.py

```diff
@@ -49,18 +55,39 @@ def create(
         file_type=file_type,
         file_import_options=file_import_options,
         upload_type=upload_type,
+        key=key,
     )
     general.add(task, with_commit)
     return task
 
 
+def remove_key(
+    project_id: str,
+    task_id: str,
+    with_commit: bool = False,
+) -> None:
+    task: UploadTask = get(project_id, task_id)
+    task.key = None
+    general.flush_or_commit(with_commit)
+
+
+def remove_all_keys(
+    with_commit: bool = False,
+) -> None:
+    tasks = get_all_global()
+    for task in tasks:
+        task.key = None
+    general.flush_or_commit(with_commit)
```

Comment:
getting all and only using them to update one field seems like a lot of overhead, especially on a server with long living db. We should only get the ones with a key (so the list should be reduced by quite a margin) or (definitely preferred) have a small update script that clears is 
something like 
```
UPDATE upload_task
SET key = NULL
WHERE key != NULL
```

Reply:
true, will refactor it to sql